_data/changelogs/component-input-mask.yml

```diff
@@ -2,6 +2,20 @@ title: Input mask
 type: component
 changelogURL:
 items:
+  - date: NNNN-NN-NN
+    summary: Fixed a bug that caused the hover state to adopt disabled styling.
+    summaryAdditional:
```

Comment:
Should we remove empty items? I believe other PRs exclude `summaryAdditional` when it is not needed.

```suggestion

```